README.md

```diff
@@ -164,6 +164,10 @@ Find the reclamation ID of the Secrets Manager service. Use that ID in the follo
 ibmcloud resource reclamation-delete <reclamation-id>
 ```
 
+### Why does the deployment fail when creating pipeline properties?
+
+This particular problem can occur when your ALM/toolchain deployment is more than 14 days old and the application configuration DA has been undeployed/redeployed. This is due to the Continuous Delivery service being required to create and delete pipeline properties, and the deployment occurs when the CD service may not exist. We are working on a long-term solution for this bug, but in the meantime it can be mitigated by ensuring the existence of a CD service in the resource group where toolchains are expected to be created. 
```

Comment:
Can you include:
- The exact DA member name that is impacted
- Does it impact both variations (I assume so)?
- What is the exact error text that is seen (incase someone searches for the error text)

Reply:
Added some more details - I kept the error message short and specific, should be fairly clear.